test/built-ins/Temporal/PlainDate/prototype/toString/calendarname-undefined.js

```diff
@@ -14,12 +14,35 @@ info: |
 features: [Temporal]
 ---*/
 
+const calendarMethods = {
+  dateAdd() {},
+  dateFromFields() {},
+  dateUntil() {},
+  day() {},
+  dayOfWeek() {},
+  dayOfYear() {},
+  daysInMonth() {},
+  daysInWeek() {},
+  daysInYear() {},
+  fields() {},
+  inLeapYear() {},
+  mergeFields() {},
+  month() {},
+  monthCode() {},
+  monthDayFromFields() {},
+  monthsInYear() {},
+  weekOfYear() {},
+  year() {},
+  yearMonthFromFields() {},
+  yearOfWeek() {},
+};
+
 const tests = [
   [[], "2000-05-02", "built-in ISO"],
-  [[{ id: "custom" }], "2000-05-02[u-ca=custom]", "custom"],
-  [[{ id: "iso8601" }], "2000-05-02", "custom with iso8601 id"],
-  [[{ id: "ISO8601" }], "2000-05-02[u-ca=ISO8601]", "custom with caps id"],
-  [[{ id: "\u0131so8601" }], "2000-05-02[u-ca=\u0131so8601]", "custom with dotless i id"],
+  [[{ id: "custom", ...calendarMethods }], "2000-05-02[u-ca=custom]", "custom"],
+  [[{ id: "iso8601", ...calendarMethods }], "2000-05-02", "custom with iso8601 id"],
+  [[{ id: "ISO8601", ...calendarMethods }], "2000-05-02[u-ca=ISO8601]", "custom with caps id"],
+  [[{ id: "\u0131so8601", ...calendarMethods }], "2000-05-02[u-ca=\u0131so8601]", "custom with dotless i id"],
```

Comment:
We allow non-ASCII calendar IDs? @sffc is this OK?

Reply:
I don't think we allow them as such - but we can't really stop someone from doing something like
```js
get id() {
  return "🍕";
}
```
I believe we previously chose not to prevent `toString` from being run on a date with such a calendar. I'll try to look for the issue or meeting note where this was discussed. (My feeling is that if you're going to use a malformed calendar ID string, you'll get a malformed ISO string, but it's probably useful to still attempt to serialize a Temporal object to a string if we can at all do so.)

Reply:
See https://github.com/tc39/proposal-temporal/issues/1815
I didn't link a discussion when I closed it, but in the meeting notes I see that we did a bugtracker sweep in the champions meeting just before I closed it, so I assume that's why.